README.md

```diff
@@ -112,7 +112,7 @@ const wss = new WebSocket.Server({
     // Other options settable:
     clientNoContextTakeover: true, // Defaults to negotiated value.
     serverNoContextTakeover: true, // Defaults to negotiated value.
-    clientMaxWindowBits: 10,       // Defaults to negotiated value.
+    clientMaxWindowBits: 10,       // Defaults to negotiated value. Right now, setting this causes conection to fail in FF.
```

Comment:
I think it's better to completely remove `clientMaxWindowBits` from the example. There is nothing wrong with FF, it's the configuration that makes the offer sent by FF unacceptable.

We already have this sentence "See docs for more options." before the example so that should be sufficient.

Reply:
i agree removing is better.